dataconf/main.py

```diff
@@ -44,15 +51,31 @@ def dict(self, obj: str, **kwargs) -> "Multi":
         conf = ConfigFactory.from_dict(obj)
         return Multi(self.confs + [conf], self.strict, **kwargs)
 
-    def string(self, s: str, **kwargs) -> "Multi":
+    def string(self, s: str, loader: str = HOCON, **kwargs) -> "Multi":
+
+        if loader == YAML:
+            data = safe_load(s)
+            return self.dict(data, **kwargs)
+
         conf = ConfigFactory.parse_string(s)
         return Multi(self.confs + [conf], self.strict, **kwargs)
 
-    def url(self, uri: str, **kwargs) -> "Multi":
-        conf = ConfigFactory.parse_URL(uri)
+    def url(self, uri: str, timeout: int = 10, **kwargs) -> "Multi":
```

Comment:
why timeout?

Reply:
Two main reasons : 
- good practice to always use a timeout when loading a remote resource, networks can be very capricious
- pyhocon `parse_url` uses a timeout, which pushes us to keep the behaviour consistent between loaders